### ISSUE TITLE
Improved UX for starting 1:1 convos

### ISSUE THREAD

**Issue by ara4n** [vector-im/vector-web#1392]
TheManiac says:
```
hmm.. in vector it occurs to me it's much less clear how to start a private discussion
if I click on start chat, it creates a public room
and if I look at my historical rooms, I can't reopen a private discussion that I had before
(Fair enough if I can't join that version of the discussion, which probably doesnt' exist any more, but it could give me the option to clone that room with the same members & privacy settings etc)
(I realise this is probably a well worn discussion/argument)
and down the right hand side is everyone in the world - the UI doesn't make it clear that if I searched for them whether I'd be inviting them to the current room, or be given a chance to start a new one
```

**Comment by dbkr** [comment]
See also https://github.com/vector-im/vector-web/issues/1613

**Comment by ara4n** [comment]
this is solved by #1882
